src/components/OpenWith/OpenWithButton.js

```diff
@@ -69,13 +70,14 @@ const OpenWithButton = ({
             position="bottom-center"
         >
             <Button
-                data-attribute-id={id}
                 isDisabled={isDisabled}
-                onClick={onClick}
+                onClick={onClick ? () => onClick(id) : noop}
```

Comment:
Typically you want to avoid inline arrow functions, as a new function is created on each render, thus the sub components will be rendered. It probably won't make much of a difference though